packages/xo-server-backup-reports/src/index.js

```diff
@@ -428,6 +426,11 @@ class BackupReportsXoPlugin {
         continue
       }
 
+      if (!force && taskLog.status === 'success' && reportWhen === 'failure') {
+        ++nSuccesses
```

Comment:
Does that mean it's not incremented when `reportWhen === 'always'`? :thinking:

Reply:
It be will incremented in the right place when the reportWhen is `always`. See https://github.com/vatesfr/xen-orchestra/blob/de98ac728c7e110c8b4c0abfc91b8d219b43a28a/packages/xo-server-backup-reports/src/index.js#L568

Reply:
I don't understand this code, IMHO, it's getting more and more like spaghetti.

I take your word that this PR is correct, we'll clean this plugin one day.